### ISSUE TITLE
Apply proper roles to status messages 

### ISSUE THREAD

**Issue by adriannelee** [cds-snc/notification-planning#496]
Issue group: Banners
Visibility: 3
Evaluation: Does not support
Success Criteria: 4.1.3: Status Messages
Needs design: 

### Description
Warning/Confirmation messages should have the [alert](https://www.w3.org/TR/wai-aria-1.1/#alert) role

![](https://dl.airtable.com/.attachments/b689397be7bfa9a08b540aad9225284d/7342d097/Capturedecranle2022-03-15a14_01_51.png)

**Comment by yaelberger-commits** [comment]
Hey team! Please [add your planning poker estimate](https://app.zenhub.com/workspaces/Notify-Planning-614b3ad91bc2030015ed22f5/issues/cds-snc/notification-planning/496?planning-poker) with ZenHub @amazingphilippe @jimleroyer @sastels

**Comment by yaelberger-commits** [comment]
Phil got some mixed results, did what was recommended in the audit. It's better, not great.
